src/components/modal/index.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import {
+    Modal as ChakraModal,
+    ModalOverlay,
+    ModalContent,
+    ModalHeader,
+    ModalFooter,
+    ModalBody,
+    Text,
+    Spacer,
+} from "@chakra-ui/react";
+import { Button } from "..";
+
+export interface ModalProps {
+    isOpen: boolean;
+    onCancel: () => void;
+    onConfirm: () => void;
+    header: string;
+    bodyText?: string;
+    cancelText?: string;
+    confirmText?: string;
+    alignButtonsRight?: boolean;
+    confirmButtonColor?: string;
+    size?: string; // "xs", "sm", "md", "lg", "xl", "full"
+}
+
+// to control modals:
+// const { isOpen, onOpen, onClose } = useDisclosure();
+
+export const Modal: React.FC<ModalProps> = (props) => {
+    const {
+        isOpen,
+        onCancel,
+        onConfirm,
+        header,
+        bodyText,
+        cancelText = "Cancel",
+        confirmText = "Confirm",
+        alignButtonsRight = true,
+        confirmButtonColor = "black",
+        size = "md",
```

Comment:
Rather than having only a few options to override like size, consider adding a `...rest` to forward props to `ChakraModal` in case we need to change the props here https://chakra-ui.com/docs/overlay/modal#props.

Reply:
great point!